pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/simple/PersistentQueue.java

```diff
@@ -110,6 +110,27 @@ public void add(PersistentQueueElement element) throws SQLException, Interrupted
         }
     }
 
+    public void addWithoutCommit(PersistentQueueElement element) throws SQLException, InterruptedException {
```

Comment:
Add a comment that explains the purpose of this method.

Reply:
Done